ConsumerApi.Sdk/Endpoints/Common/Types/CollectionResponseBase.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections;
+
+namespace Backbone.ConsumerApi.Sdk.Endpoints.Common.Types;
+
+public abstract class CollectionResponseBase<TItem> : ICollection<TItem>
```

Comment:
I just realized that I made a mistake here. Can you replace this with the code you find here:

https://github.com/nmshd/backbone/blob/main/BuildingBlocks/src/BuildingBlocks.Application/CQRS/BaseClasses/CollectionResponseBase.cs

And also rename it to "EnumerableResponseBase".

Reply:
👍